packages/expo-build-properties/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "expo-build-properties",
+  "version": "0.1.0",
+  "description": "Config plugin to customize native build properties on prebuild",
+  "main": "build/withBuildProperties.js",
+  "types": "build/withBuildProperties.d.ts",
+  "scripts": {
+    "build": "expo-module build",
+    "clean": "expo-module clean",
+    "lint": "expo-module lint",
+    "test": "expo-module test",
+    "prepare": "expo-module prepare",
+    "prepublishOnly": "expo-module prepublishOnly",
+    "expo-module": "expo-module"
+  },
+  "keywords": [
+    "react-native",
+    "expo",
+    "build",
+    "build-properties"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/expo/expo.git",
+    "directory": "packages/expo-build-properties"
+  },
+  "bugs": {
+    "url": "https://github.com/expo/expo/issues"
+  },
+  "author": "650 Industries, Inc.",
+  "license": "MIT",
+  "homepage": "https://docs.expo.dev/versions/latest/sdk/build-properties",
+  "dependencies": {
+    "@expo/config-plugins": "^4.1.2",
+    "ajv": "^8.11.0"
+  },
+  "devDependencies": {
+    "expo-module-scripts": "^2.0.0"
+  },
+  "jest": {
+    "preset": "expo-module-scripts"
+  },
+  "peerDependencies": {
+    "expo": "*"
```

Comment:
@brentvatne we should ensure this package is added to the versions endpoint and probably also the `expo/bundledNativeModules.json` so it works with `expo install`.

Reply:
added in https://github.com/expo/expo/pull/17106/files#diff-b06ae9dbb36aa269c31f8467b209196d26ae729dfaa5afe21cdd8d50066814dfR30